### ISSUE TITLE
Improve support for .org files

### ISSUE THREAD

**Issue by lepisma** [isaacs/github#798]
Github's current previews for Emacs [org mode](http://orgmode.org/) files are pretty basic. Its okay if the files are having elements transformable to markdown, but it gets real hairy with many org specific features like
- Todo tags `TODO`, `DONE` etc.
- General tags `:tag:`
- `DEADLINE`, `SCHEDULE` and other time related properties
- Drawers etc.

These are really useful for people who plan projects with org files. I don't know where this lies as far as the priority order is concerned, but it will be useful to have a *complete* org preview which can help people browsing around without Emacs (I mean, that's what the web preview is for).

**Comment by lepisma** [comment]
From: GitHub Staff <support@github.com>
To: "Abhinav Tushar @ gmail" <abhinav.tushar.vs@gmail.com>
Subject: Re: Feature request: Better .org previews
Date: Sun, 23 Oct 2016 05:56:15 -0700

Hi Abhinav,

Thanks for your suggestion! I can't promise we'll do it, but I have added it to our list for our team to consider!

**Comment by TimoKramer** [comment]
Is it possible to have badges in org-mode rendered on github in a README.org? I was trying some things but it was never rendered:(

**Comment by lepisma** [comment]
yes, see [this](https://github.com/lepisma/org-krita) for example.
